### PR TITLE
Optimize the sum operation

### PR DESCRIPTION
**Pull request by sumleo**
Refine the code with directly add operation
#### Changes proposed in this Pull Request

* Replace the summation function by the lodash sum

#### Testing instructions
Run the unit test

*

### CODE REVIEW

File: client/state/jetpack-sync/selectors.js [hunk 1]
```diff
@@ -99,23 +99,23 @@ function getSyncProgressPercentage( state, siteId ) {
 	const countQueued = reduce(
```
Comment (sirreal) [inline client/state/jetpack-sync/selectors.js]: How about just using lodash's `sum`?

```suggestion
	const countQueued = sum( queued );
```

File: client/state/jetpack-sync/selectors.js [hunk 2]
```diff
@@ -99,23 +99,23 @@ function getSyncProgressPercentage( state, siteId ) {
...
 		queued,
```
Comment (sirreal) [inline client/state/jetpack-sync/selectors.js]:
```suggestion
```

File: client/state/jetpack-sync/selectors.js [hunk 3]
```diff
@@ -99,23 +99,23 @@ function getSyncProgressPercentage( state, siteId ) {
...
 		( sum, value ) => {
```
Comment (sirreal) [inline client/state/jetpack-sync/selectors.js]:
```suggestion
```

File: client/state/jetpack-sync/selectors.js [hunk 4]
```diff
@@ -99,23 +99,23 @@ function getSyncProgressPercentage( state, siteId ) {
...
-			return ( sum += value );
+			return ( sum + value );
```
Comment (sirreal) [inline client/state/jetpack-sync/selectors.js]:
```suggestion
```

File: client/state/jetpack-sync/selectors.js [hunk 5]
```diff
@@ -99,23 +99,23 @@ function getSyncProgressPercentage( state, siteId ) {
...
 		},
```
Comment (sirreal) [inline client/state/jetpack-sync/selectors.js]:
```suggestion
```

File: client/state/jetpack-sync/selectors.js [hunk 6]
```diff
@@ -99,23 +99,23 @@ function getSyncProgressPercentage( state, siteId ) {
...
 		0
```
Comment (sirreal) [inline client/state/jetpack-sync/selectors.js]:
```suggestion
```

File: client/state/jetpack-sync/selectors.js [hunk 7]
```diff
@@ -99,23 +99,23 @@ function getSyncProgressPercentage( state, siteId ) {
...
 	);
```
Comment (sirreal) [inline client/state/jetpack-sync/selectors.js]:
```suggestion
```

**Comment by sirreal** [comment]
I've attempted to provide a multiline suggested change which is pretty mangled by Github, sorry for the noisy suggestion.

**Comment by sumleo** [comment]
Should I open another PR? Or just commit again?

**Comment by sirreal** [comment]
> Should I open another PR? Or just commit again?

You can commit on this branch and edit the PR description here. There's no need to create a new PR.

**Pull request closed without merging**